service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -83,18 +83,27 @@ public WorkflowQueryResult runSummary(String runId) throws ApiException {
 
   public Object getOutputs(String id)
       throws ApiException, DependencyNotAvailableException, AzureAccessTokenException {
-    return cromwellClient.wesAPI().getRunLog(id).getOutputs();
+    ApiClient client = cromwellClient.getWriteApiClient();
+
+    return cromwellClient.wesAPI(client).getRunLog(id).getOutputs();
   }
 
-  public WorkflowDescription describeWorkflow(String workflowUrl) throws ApiException {
-    return cromwellConfig.womtoolApi().describe(API_VERSION, null, workflowUrl, null, null, null);
+  public WorkflowDescription describeWorkflow(String workflowUrl)
+      throws ApiException, DependencyNotAvailableException, AzureAccessTokenException {
+    ApiClient client = cromwellClient.getWriteApiClient();
+    return cromwellClient
+        .womtoolApi(client)
+        .describe(API_VERSION, null, workflowUrl, null, null, null);
   }
 
-  public String getRunErrors(Run run) throws ApiException {
+  public String getRunErrors(Run run)
+      throws ApiException, DependencyNotAvailableException, AzureAccessTokenException {
+
+    ApiClient client = cromwellClient.getWriteApiClient();
```

Comment:
I think here also we might want to use the read client since we are getting updates related to workflow?